lib/prmd/templates/schemata/link.md.erb

```diff
@@ -62,9 +62,9 @@ HTTP/1.1 <%=
 <%- end %>
 ```
 
-<%- if response_example || link['rel'] != 'empty' %>
+<%- if response_example.has_key?('body') || link['rel'] != 'empty' %>
 ```json
-<%- if response_example %>
+<%- if response_example.has_key?('body') %>
```

Comment:
Also, we should probably change this to just have the whole thing if it doesn't have body maybe? I would hate to break backwards compatibility anyway if it can be avoided.